test-poc/framework/src/main/java/org/keycloak/test/framework/database/TestDatabase.java

```diff
@@ -24,7 +32,19 @@ public TestDatabase(DatabaseConfig databaseConfig) {
     public void start() {
         if (databaseConfig.getContainerImage() != null) {
             container = createContainer();
-            container.withStartupTimeout(Duration.ofMinutes(5)).start();
+            container.withStartupTimeout(Duration.ofMinutes(30))
+                    .withLogConsumer(logConsumer)
+                    .withEnv(databaseConfig.getEnv())
+                    .start();
+            try {
+                if (databaseConfig.getPostStartCommand() != null) {
+                    LOGGER.info("Running post start command: " + databaseConfig.getPostStartCommand());
```

Comment:
Use tracev or tracef instead

Reply:
Do you want to use `jboss-logging` instead of `slf4j-api`?